modules/ztp-install-factory-pipeline.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: PROCEDURE
+[id="run-the-hub-cluster-factory-install-pipeline_{context}"]
+= Run the hub cluster factory install pipeline
+
+Follow the guidance here to run the hub factory install pipeline.
+
+.Prerequisites
+
+* An {product-title} hub cluster.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a file `spokes.yaml` with sample details as shown. A sample configuration file is present in `examples/config.yaml`.
++
+[NOTE]
+====
+At this stage you only need to build out the `config` section. The `config` section specifies the cluster configuration values used to install and configure the hub and edge cluster.
+====
++
+[source,yaml]
+----
+config:
+  OC_OCP_VERSION: "4.10.9" <1>
+  OC_ACM_VERSION: "2.4" <2>
+  OC_OCS_VERSION: "4.9" <3>
+----
++
+<1> {product-title} version of the edge cluster.
+<2> Red Hat Advanced Cluster Management (RHACM) version.
+<3> The OpenShift Data Foundation (ODF) version.
+
+. Start the hub cluster pipeline from the command line:
++
+[source,terminal]
+----
+$ tkn pipeline start -n spoke-deployer -p git-revision=main spokes-config="$(cat /path-to-spoke.yaml/spokes.yaml)" -p kubeconfig=${KUBECONFIG} -w=ztp,claimName=ztp-pvc --timeout 5h --use-param-defaults deploy-ztp-hub
```

Comment:
we don't use -p git-revision=main anymore

Reply:
removed